sailthru_content/services/sailthru_translation_service.py

```diff
@@ -58,11 +60,23 @@ def _create_course_vars(self, course, course_run, url, site_name, program_dictio
             for seat in course_run['seats']:
                 sailthru_content_vars['price_{}'.format(seat['type'])] = seat['price']
                 sailthru_content_vars['currency_{}'.format(seat['type'])] = seat['currency']
+                if not sailthru_content_vars.get('course_type', None):
+                    sailthru_content_vars['course_type'] = seat['type']
+                else:
```

Comment:
Are you thinking that we need to update the existing items in the Sailthru content library this way? We don't need to do that. You can remove this `else` clause